src/renderer/components/UserFeedback/ProvideFeedbackModal.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box, Typography, Stack, styled, Modal } from '@mui/material';
+import { useState } from 'react';
+import colors from 'renderer/colors';
+import {
+  PrimaryButton,
+  SecondaryButton,
+} from 'renderer/components/Blocks/Buttons';
+import ipc from 'renderer/ipc';
+import { URL_USER_FEEDBACK_FORM } from '../../../constants';
+import ReportBugModal from './ReportBugModal';
+
+const CustomStack = styled(Stack)`
+  width: 100%;
+  height: 100%;
+`;
+
+const CustomModal = styled(Modal)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const CustomModalInner = styled(Box)`
```

Comment:
Nitpick but we're generally using objects to define styles rather than template literals so that is preferred, it's not a huge deal though